packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -487,6 +481,10 @@ const renderer =
                                     )
                                     .replace(/\\/g, '/'),
                                 to: `static/`
+                            },
+                            {
+                                from: path.resolve('translations/compiled'),
+                                to: 'static/translations/compiled/'
```

Comment:
A small downside of this approach is that previously, i18n is a feature of the retail react app, not the sdk. But now with this change, the implementation for the feature is leaked across packages. Not a deal breaker just wanna call it out.

Reply:
I also have that idea but i'm not sure how much effort is that

Reply:
@kevinxh It is relatively easy, we just need to change the output for compiled commands to static folder instead of translation folder

Reply:
Note: what we wanted to try turns out to be a bit more complex than what I would have liked. The [compile script](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c608b90aaf2278575566b326811bf76c2fc67f6e/packages/template-retail-react-app/package.json#L15) does not know whether the generated project would have an overrides folder or not (in order to target “overrides/app/static” folder)

Reply:
@alexvuong given my comment above, the PR as it is now will still avoid running into the duplicate translation issue you were thinking of. I've checked the `build` folder and `npm run analyze-build`.

Reply:
How about writing a custom scripts to handle that case? A bit more maintaining debt on our side, but it will be in the template, and not leak out to pwa-kit-dev. 🤔